src/modules/deal-investor/dto/get-deals-investors-params.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { UkefId } from '@ukef/helpers';
+import { Matches } from 'class-validator';
+
+export class GetDealsInvestorsParamsDto {
+  @ApiProperty({ description: 'UKEF id for deal', example: '0030000321' })
+  @Matches(/00\d{8}/)
```

Comment:
Since all the IDs are 8 digit can we please pick this value from a constant?

Reply:
Not sure if this is about Regex or example.
Left RegEx as is, we will do more work on Ids later, maybe custom decorator.
Changed example to constant.

Reply:
Regarding the RegEx, I discovered the `@IsNumberString()` decorator this afternoon. A combination of this and `@Length(8, 8)` might result in more readable code and more descriptive error messages, but this point is common to several tickets so can be worried about later (e.g., in its own ticket) rather than this PR

Reply:
To make this validation reusable we can create our own decorator for ukefId.
Current regex is more strict because id should start with 00. Not sure if this is too much, but it matches real live data